src/sql/magic.py

```diff
@@ -436,6 +453,18 @@ def interactive_execute_wrapper(**kwargs):
                 if detailed_msg is not None:
                     err = exceptions.UsageError(detailed_msg)
                     raise err
+                    # TODO: move to error_messages.py
```

Comment:
looks like there is some pending work here

Reply:
I'm not able to move this logic to `error_messages.py` because of the circular dependency mentioned [here](https://github.com/ploomber/jupysql/pull/486#issuecomment-1559645056)

Reply:
ok, can you open an issue? we need to do some refactoring here.

I was unable to find a fix because the traceback in the link you shared is outdated, seems like some imports already changed, so let's create an issue and I can review this later.

Reply:
Opened [issue](https://github.com/ploomber/jupysql/issues/545)

Reply:
Added issue numbers